scripts/eosio_build_centos.sh

```diff
@@ -493,224 +495,204 @@ mongodconf
 		fi
 	fi
 
-    if [ $MONGO_INSTALL == "true" ]; then
-		if ! cd "${TEMP_DIR}"
-		then
-			printf "\\tUnable to enter directory %s.\\n" "${TEMP_DIR}"
-			printf "\\tExiting now.\\n\\n"
+	if [ $MONGO_INSTALL == "true" ]; then
+		if ! cd "${TEMP_DIR}"; then
+			printf "\\t!! Unable to enter directory %s !!\\n" "${TEMP_DIR}"
+			printf "\\tExiting now.\\n"
 			exit 1;
 		fi
-		STATUS=$( curl -LO -w '%{http_code}' --connect-timeout 30 https://github.com/mongodb/mongo-c-driver/releases/download/1.10.2/mongo-c-driver-1.10.2.tar.gz )
+		MONGODRIVERTGZ="mongo-c-driver-1.10.2.tar.gz"
+		MONGODRIVEURL="https://github.com/mongodb/mongo-c-driver/releases/download/1.10.2/${MONGODRIVERTGZ}"
+		STATUS=$( curl -LO -w '%{http_code}' --connect-timeout 30 "${MONGODRIVERURL}" )
 		if [ "${STATUS}" -ne 200 ]; then
-			if ! rm -f "${TEMP_DIR}/mongo-c-driver-1.10.2.tar.gz"
-			then
-				printf "\\tUnable to remove file %s/mongo-c-driver-1.10.2.tar.gz.\\n" "${TEMP_DIR}"
+			if ! rm -f "${TEMP_DIR}/${MONGODRIVERTGZ}"; then
+				printf "\\t!! Unable to remove file %s/${MONGODRIVERTGZ} !!\\n" "${TEMP_DIR}"
 			fi
-			printf "\\tUnable to download MongoDB C driver at this time.\\n"
-			printf "\\tExiting now.\\n\\n"
+			printf "\\t!! Unable to download MongoDB C driver at this time !!\\n"
+			printf "\\tExiting now.\\n"
 			exit 1;
 		fi
-		if ! tar xf mongo-c-driver-1.10.2.tar.gz
-		then
-			printf "\\tUnable to unarchive file %s/mongo-c-driver-1.10.2.tar.gz.\\n" "${TEMP_DIR}"
-			printf "\\tExiting now.\\n\\n"
+		if ! tar xf "${MONGODRIVERTGZ}"; then
```

Comment:
it seems highly likely that this line has been bad for sometime and is now causing CI to reject your changes.  Should it be `tar xf "${TEMP_DIR}/${MONGODRIVERTGZ}"` ?

Reply:
Ah, good catch. I'll pwd in there and see what directory I'm in when I extract the tar. I'll confirm later tonight

Reply:
`pwd` shows I'm in the tmp directory when it gets to tar xf, so we're good there. Further investigation reveals MONGODRIVEURL was used for STATUS=$() and not MONGODRIVERURL, with an R.